recipes/pymultinest/meta.yaml

```diff
@@ -0,0 +1,48 @@
+package:
+  name: pymultinest
+  version: "2.6"
+
+source:
+  fn: pymultinest-2.6.tar.gz
+  url: https://pypi.python.org/packages/c2/fc/77bdd8356068125bba0a5c07bfa2b4a3f492bff614aeae783bc1a566d699/pymultinest-2.6.tar.gz
+  md5: d9e5eee6abd3784056ca5ccc1f05cab9
+
+build:
+   number: 1
+   skip: true  # [win]
+   skip: true  # [py3k]
+
+requirements:
+
+  build:
+    - toolchain
+    - python x.x
+    - setuptools
+    - multinest
+    - numpy
+    - scipy
+    - matplotlib
+    - pytest
+
+  run:
+    - python x.x
+    - multinest
+    - numpy
+    - scipy
+    - matplotlib
+    - pytest
+
+test:
+  imports:
+    - pymultinest
+
+about:
+  home: http://johannesbuchner.github.com/PyMultiNest/
+  license: GPLv3
```

Comment:
please manually package the `license_file`:

https://conda-forge.org/docs/meta.html#packaging-the-license-manually

and add an issue/PR upstream, adding the url to that issue as a comment in the `meta.yaml`

Reply:
ok, I followed those instructions, thanks!

Reply:
Please link the issue/PR that you opened upstream. Also please add it in a comment above the `license_file` in the `meta.yaml`.